.aspect/workflows/config.yaml

```diff
@@ -48,6 +48,8 @@ workspaces:
 tasks:
     warming:
     branch_freshness:
+        update_strategy: rebase
```

Comment:
These can be removed really as they are all the defaults, you get the `warming` and `branch_freshness` tasks for free.

Reply:
default for branch_freshness is `rebase`?

Reply:
Yup